internal/controller/instaslice_daemonset.go

```diff
@@ -353,7 +343,7 @@ func (r *InstaSliceDaemonsetReconciler) Reconcile(ctx context.Context, req ctrl.
 				//TODO: could be merged with the deleted call below
 				err := r.Get(ctx, typeNamespacedName, &updateInstasliceObject)
 				if err != nil {
-					log.FromContext(ctx).Error(err, "error getting latest instaslice object")
+					return ctrl.Result{RequeueAfter: 1 * time.Second}, nil
```

Comment:
err not returned
1 second requeue seems fast... The time duration should be a constant

Reply:
I will make the time value constant. daemonset should be ahead of the controller to process allocations.